Info/handler.ts

```diff
@@ -25,7 +25,7 @@ type InfoHandler = () => Promise<
 
 type HealthChecker = (
   config: unknown
-) => healthcheck.HealthCheck<healthcheck.ProblemSource, true>;
+) => healthcheck.HealthCheck<"AzureStorage" | "Config" | "AzureCosmosDB", true>;
```

Comment:
ditto

Reply:
Yess this is related to the `@pagopa/io-functions-commons` update